Changelog.md

```diff
@@ -14,6 +14,9 @@ rules on making a good Changelog.
 
 - Improved error message for when a MacOS target version is not met.
   [#211](https://github.com/matthew-brett/delocate/issues/211)
+- Retag wheels automatically when fusing to ensure the wheel name and WHEEL
+  file tag data is accurate.
+  [#215](https://github.com/matthew-brett/delocate/pull/215)
```

Comment:
The changelog should mention specifically that the previous behavior of `delocate-fuse` no longer applies.
```suggestion
  `delocate-fuse` will now output to a new wheel with updated tags instead of replacing the first wheel given.
  [#215](https://github.com/matthew-brett/delocate/pull/215)
```
The readme's "Making dual-architecture binaries" section also needs to be updated. Luckily this will mostly mean deleting the documented workarounds now that the command works more intuitively.

I'm not sure how much to focus on documenting the change in the `fuse_wheels` function. It feels awkward to track changes for `0.x.x` versions. Normally you'd append a note to the docstring like this:
```py
"""
...

.. versionchanged:: 0.12
    `out_wheel` can now take a directory or None.
"""
```

Reply:
- [x] Changelog
- [x] README
- [ ] Docstring, I updated it the way I think I was supposed to. But you can double check this and see if its to your liking.

Reply:
The docstring looks okay.